brave/browser-laptop
# New database for verifying publishers

Submitter Checklist:


Resolves #11678

-----------
- [x] Plan A:
- create wallet on the latest master
- switch to this branch
- check that folder `ledger-publishersV2.leveldb` is deleted in the user profile
-----------
- [x] Plan B (publisher is not verified):
- create wallet
- visit https://slo-tech.com and stay on that site for 15s
- go back to ledger table and verify that there is no verify icon
-----------
- [x] Plan C (publisher is verified, but didn't opt-in display):
- create wallet
- visit https://clifton.io and stay on that site for 15s
- go back to ledger table and verify that there is no verify icon
-----------
- [x] Plan D (publisher is verified and did opt-in display):
- create wallet
- visit https://duckduckgo.com/ and stay on that site for 15s
- go back to ledger table and verify that there is verify icon
-----------
Reviewer Checklist:

Tests
